source/quick-start.txt

```diff
@@ -0,0 +1,110 @@
+.. _entity-framework-quickstart:
+
+===========
+Quick Start
+===========
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. include:: /includes/quick-start/overview.rst
+
+Create a MongoDB Cluster
+------------------------
+
+Set Up a Free Tier Cluster in Atlas
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To set up your Atlas Free Tier Cluster required for this guide, complete the guide on 
+:guides:`MongoDB Atlas Setup </atlas/account>`.
+
+After completing the steps in the Atlas guide, you have a new MongoDB
+cluster deployed in Atlas, a new database user, and
+:atlas:`sample datasets loaded </sample-data/>` into your cluster. You also have
```

Comment:
Suggestion:
I think the link could be more descriptive in terms of what it's linking, or be removed since it references part of the content contained within the previous link. Here are some[ link and cross-reference examples](https://www.mongodb.com/docs/meta/style-guide/style/links-and-cross-references/#link-and-cross-reference-examples) in the style guide.